test/unit/outputs.test.ts

```diff
@@ -26,6 +27,7 @@ const ValidOutputsTestcases: ValidParseTestcase<TransactionOutput[]>[] = [
                 type: AmountType.WITHOUT_MULTIASSET,
                 coin: toUint("18446744073709551616"),
             },
+            datumHash: undefined,
```

Comment:
If we did not add this here, won't `datumHash` be undefined anyway?

Reply:
It will, but `expect(parsed).to.deep.equal(expectedParsed)` distinguishes between a missing object key and a present key with value `undefined`.